### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by yn-mrse**

### CODE REVIEW

**Comment by yn-mrse** [comment]
See this [link](https://github.com/eve-autonomy/delivery_reservation_lamp_manager/blob/816b6d540e37941b0f3147388310602cdaa4bfa1/README.md) for node graphs.

File: README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ The lighting pattern is defined as follows.
 
 ## Input and Output
 - input
-  - from eve oss
+  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)
     - `/autoware_state_machine/lock_state` : Reservation status for on-demand delivery.
 - output
-  - to tier iv oss
+  - to [dio_ros_driver](https://github.com/tier4/dio_ros_driver)
     - `/dio/dout3` : GPIO output topic. (this topic remapping from /delivery_reservation_lamp_out)
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
    - `/dio/dout3` : Digital-out assignment to a 3rd pin in [0-7] general-purpose outputs. This topic is remapped from `/delivery_reservation_lamp_out`.
```

And, add a description about values true/false, 1/0, against to the output from the /dio_ros_driver node.

File: README.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ The lighting pattern is defined as follows.
 
 ## Input and Output
 - input
-  - from eve oss
+  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)
     - `/autoware_state_machine/lock_state` : Reservation status for on-demand delivery.
```
Comment (MasaakiWatanabeTier4) [inline README.md:33]: Add a description about values true/false, 1/0, against to the input from the /autoware_state_machine node.

**Review by MasaakiWatanabeTier4 — COMMENTED** [review]
I left three comments.

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
 # delivery_reservation_lamp_manager
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
# Lamp manager for delivery reservation
```

File: README.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+
+## Overview
+By changing the lighting pattern of the lamp, this node notifies senders of the on-demand delivery reservation status.
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
By changing lighting pattern of the lamp, this node notifies surrounding workers of on-demand delivery reservation status through `/dio_ros_driver`. 
```

File: README.md [hunk 5]
```diff
@@ -1 +1,42 @@
 # delivery_reservation_lamp_manager
+
+## Overview
+By changing the lighting pattern of the lamp, this node notifies senders of the on-demand delivery reservation status. 
+
+The lighting pattern is defined as follows.
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
Lighting patterns are listed below.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,42 @@
...
+  <thead>
+    <tr>
+      <th scope="col">Reservation status</th>
+	    <th scope="col">lighting pattern</th>
+	  </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Not reserved</td>
+      <td>Keeps going off</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>Off</td>
```

File: README.md [hunk 7]
```diff
@@ -1 +1,42 @@
...
+    </tr>
+    <tr>
+      <td>In progress</td>
+      <td>Blinks</td>
+    </tr>
+    <tr>
+      <td>Reserved</td>
+      <td>Keeps lighting</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>Light up</td>
```

**Review by MasaakiWatanabeTier4 — CHANGES_REQUESTED** [review]

**Pull request merged**